app/recipe/views.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import viewsets, mixins
+
+from core.models import Recipe, Ingredient
+
+from recipe import serializers
+
+
+class IngredientViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
```

Comment:
This viewset is not required for the API to work as expected, so you can delete it.

Minor thing: the mixins are usually placed to the left of the base class in the inheritance path, so if you wanted this it would be more like:
```
IngredientViewSet(mixins.ListModelMixin, viewsets.GenericViewSet)
```